test/erlfmt_SUITE.erl

```diff
@@ -1191,9 +1200,9 @@ assert_snapshot_match(Expected, Output) ->
         {skip, _} ->
             ok;
         {ok, Other, _} ->
-            % Split by lines.
-            Expected2 = string:lexemes(Expected, [$\n]),
-            Other2 = string:lexemes(Other, [$\n]),
+            % Split by lines (preserving empty lines).
+            Expected2 = string:split(Expected, "\n", all),
```

Comment:
Not so familiar with the difference between `lexemes` and `split` how did you decide that this change was important?

Reply:
Oh I see

![Screen Shot 2021-05-27 at 12 01 05 PM](https://user-images.githubusercontent.com/1829933/119815233-3cb0b780-bee3-11eb-860a-166f307e38e5.png)

Was this the reason?

Reply:
The main reason is to preserve empty lines:
```
% Line 1
% Line 2

% Line 4
```
With lexemes, `% Line 4` would become the 3rd item of the list. So, in case of mismatch error, the index wouldn't correspond to the line number, which is confusing.